org.kde.kamoso.json

```diff
@@ -0,0 +1,55 @@
+{
+    "id": "org.kde.kamoso",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kamoso",
+    "rename-icon": "kamoso",
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=host",
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=x11",
+        "--talk-name=org.a11y.Bus"
+    ],
+    "cleanup": [
+        "*.a",
+        "*.la",
+        "/include",
+        "/lib/pkgconfig",
+        "/share/doc",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "kamoso",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.0/src/kamoso-21.12.0.tar.xz",
+                    "sha256": "0d35986fb1a093023f2242b54a4db1e7ce0e6a91eac51dbe29f910c37e993d3a"
+                },
+                {
+                    "type": "patch",
+                    "path": "appdata.patch"
+                }
+            ]
+        },
+        {
+            "name": "missing-icon",
```

Comment:
you don't need a new module for that, it's confusing. Just add the install in the post-install for the main app, and the icon in its source section.

Reply:
also install it as "kamoso.png" since you have a `rename-icon` at the top.